.gitignore

```diff
@@ -1,2 +1,160 @@
-.env*
```

Comment:
You are removing these two lines, which 1) means someone can accidentally push to github any extended .env they might be using, and 2) .idea files will now also show up in the github.

Please, update your changes to keep these

Reply:
`.env` is covered in this complete .gitignore. I will uncomment the `.idea/`at the end.

Reply:
Umh, the reason we have it as .env* is so that people can also have .env.myfeature and have it ignore. There's at least one person who's using this already.

Reply:
Gotcha, i wasn't aware of that feature. It seems like we prefer building this as we go anyways. I'll cancel the PR